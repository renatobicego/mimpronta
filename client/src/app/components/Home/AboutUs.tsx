import Image from 'next/image'
import Link from 'next/link'
import React from 'react'

const AboutUs = () => {
  return (
    <section className='mt-0 sm:mt-[10vh] w-[80dvw] lg:w-[75dvw] flex flex-col items-center gap-4 relative'>
        <h2 role='definition' className='phrase-size text-center leading-phrase font-semibold md:w-[75%] font-title mb-4' >
            Transformamos habilidades en <span className='font-extrabold'>propósitos</span> e
            intenciones en <span className=' font-extrabold'>acciones</span>, ayudando a generar un 
            <span className='text-azul font-extrabold'> impacto</span> social positivo. 
        </h2>
        <p role='definition' className='paragraph-size leading-8 lg:leading-10 text-center font-semibold md:w-3/4 list-none'>
          Combinamos una impronta de negocios con una impronta social, facilitando 
          herramientas prácticas de 
          <span className='font-extrabold'> Innovación, Marketing Responsable y Sostenibilidad, </span>
          para lograr una entrega de valor combinado. 
        </p>
        <Link href={'/sobreMimpronta'}>
          <button role='button' className='btn-primary mt-4 '>
              Sobre mimpronta
          </button>
        </Link>
    </section>
  )
}

export default AboutUs