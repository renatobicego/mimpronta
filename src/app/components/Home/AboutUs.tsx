import React from 'react'

const AboutUs = () => {
  return (
    <section className='mt-[10vh] lg:mt-[15vh] w-[80dvw] lg:w-[60dvw] flex flex-col items-center gap-4'>
        <p role='definition' className='paragraph-size text-center font-semibold md:w-[70%]'>
            Transformamos habilidades en propósitos, intenciones en acciones, que ayuden a generar un impacto social positivo. 
        </p>
        <p role='definition' className='paragraph-size text-center font-semibold md:w-3/4'>
            Combinamos una impronta de negocios, con una impronta social, facilitando herramientas 
            prácticas de Innovación, Marketing, Responsabilidad social corporativa (RSC) y Sostenibilidad, 
            para lograr una entrega de valor combinado. 
        </p>
        <button role='button' className='btn-primary mt-4 '>
            Sobre Mimpronta
        </button>
    </section>
  )
}

export default AboutUs