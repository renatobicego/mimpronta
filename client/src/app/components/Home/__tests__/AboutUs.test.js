import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom'
import AboutUs from '../AboutUs.tsx'

describe('About Us', () => {
    it('renders unchanged', () => {
        const {container} = render(<AboutUs />)
        expect(container).toMatchSnapshot()
    })

    it('renders content', async() => {
        render(<AboutUs />)
        const title = await screen.findByRole('heading')
        const paragraphs = await screen.findAllByRole('definition')
        const btnNuestroProposito = await screen.findByRole('button')

        expect(title).toBeInTheDocument()
        expect(btnNuestroProposito).toBeInTheDocument()
        expect(paragraphs).toHaveLength(2)
    })

});