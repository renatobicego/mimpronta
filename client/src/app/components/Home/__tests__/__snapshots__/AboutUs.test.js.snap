// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Us renders unchanged 1`] = `
<div>
  <section
    class="mt-[10dvh] lg:mt-[25dvh] w-[80dvw] lg:w-[60dvw] flex flex-col items-center gap-4"
  >
    <h5
      class="title-size text-center xsm:mb-4"
      role="heading"
    >
      ¿Quiénes somos?
    </h5>
    <p
      class="paragraph-size text-center md:w-3/4"
      role="definition"
    >
      Somos una consultora especializada en transformar ideas e intenciones en acciones concretas que generen un impacto social positivo.
    </p>
    <p
      class="paragraph-size text-center md:w-4/5"
      role="definition"
    >
      Combinamos una impronta de negocios, con una impronta social para lograr una entrega de valor combinado, facilitando herramientas prácticas de Innovación, Marketing, Responsabilidad Social Corporativa (RSC) y Sostenibilidad.
    </p>
    <button
      class="btn-primary mt-4 lg:mt-8"
      role="button"
    >
      Nuestro Propósito
    </button>
  </section>
</div>
`;
